src/livemaker/lsb/novel.py

```diff
@@ -430,7 +430,7 @@ class TWdOpeVar(BaseTWdGlyph):
         "unk3" / construct.If(construct.this._._.version > 100, construct.Int32ul),
         "link_name"
         / construct.If(
-            construct.this._._.version < 105 and construct.this._._.version > 100,
+            lambda this: 100 < this._._.version < 105,
```

Comment:
the evaluation order when using the `construct.this` lambda means the compound boolean `and` wasn't being evaluated as expected. It's better to just write the explicit context/this lambda expression that we want here